src/usermod.c

```diff
@@ -866,6 +878,14 @@ static void update_gshadow (void)
 			continue;
 		}
 
+		/*
+		* If rflg+Gflg  is passed in AKA -rG invert is_member, to remove targeted
+		* groups while leaving the user apart of groups not mentioned
+		*/
+		if (Gflg && rflg && ((!is_member && was_member) || (is_member && was_member)) ) {
+			is_member = !is_member;
+		}
```

Comment:
Do you want to do anything about sg_adm?

Reply:
Not sure what we want to do here. Any thoughts?

Reply:
I guess we shouldn't do anything.  Admins are tracked separately from members.